build_enhancements/shebanger.py

```diff
@@ -36,32 +40,31 @@
 #     ]
 # }
 
-class ShebangerCommand(sublime_plugin.WindowCommand):
+class ShebangerCommand(ExecCommand):
     """
     Command to be used as the "target" option in a build system. Based on a
     customized build system, this will modify the version of python used to
-    the one listed in the shebang line at the start of the script (if any).
+    the one listed in the shebang line at the start of the script(if any).
```

Comment:
methinks someone did a find and replace for "space open paren" to remove spaces :)

Reply:
Maaaaybe...